app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: str):
+    selected_cafe = Cafe(cafe)
+    some_arr = []
+    masks_to_buy = 0
+    for person in friends:
+        try:
+            selected_cafe.visit_cafe(person)
+            some_arr.append(person)
+        except VaccineError:
```

Comment:
Maybe separate messages for outdated vaccine and no vaccine at all should be more informative, but the solution is eloquent and according to terms. 😃